README.md

```diff
@@ -166,10 +166,11 @@ EveryAuth CLI and middleware communicate with the Fusebit APIs to do their job a
 
 The express middleware locates credentials in the following way, in priority order:
 
-1. Programmatically through code.
-2. JSON in the `EVERYAUTH_PROFILE_JSON` environment variable.
-3. The `EVERYAUTH_PROFILE_PATH` environment variable points to the `settings.json` file in a directory.
-4. The `settings.json` file in the `.fusebit` subdirectory of the current or closest parent directory.
+1. Use a JWT generated via `everyauth token` in the `EVERYAUTH_PROFILE_TOKEN` environment variable.
```

Comment:
Not sure we should encourage this pattern since it promotes long-lived access tokens. If folks want long-lived, they can as well export the profile.

Reply:
@bennbollay for the Vercel scenario, we can't rely on expiring access tokens, is that use case supported? 
@tjanczuk Adding long-lived token solves the situation to have a 5Kb limitation on environment variables space per deployment on Vercel, same applies to Lambda, the workaround is to use encrypted profiles, but we wanted to facilitate the process by using a profile token.

Reply:
@degrammer All tokens have an expiration time.  With this you can set it to be as long as your operational comfort allows.
@tjanczuk This isn't materially different than the very conventional security posture of including a secret token in your environment variables.  It just so happens that we support a better option which sophisticated users will appreciate, and unsophisticated users won't understand.

Reply:
At minimum, "programmatically through code" should take precedence over the environment variable, right?

Can we also rename this to EVERYAUTH_TOKEN - at this point it has nothing to do with a profile.